interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md

```diff
@@ -15,25 +15,52 @@ To add the **Picture Box** control to a report, drag the **Picture Box** item fr
 
 Specify one of the following properties to set an image:
 
-- **ImageSource**  
-    Use this property to save images along with a report definition.
-- **ImageUrl**  
-    Use this property to save only the path to the image.
+- **Image Source**  
+    Save the image to the report's definition.
+- **Image Url**  
+    Save only a path to the image.
 
 ![](../../../../../images/eurd-win-picture-box-image-property.png)
 
+## Assign an External Image
+
+Click the **Image Source** / **Image URL** property's ellipsis button and the **Open File** dialog is invoked.
+
+![](../../../../../images/eurd-win-ImageSource-OpenFileDialog.png)
+
+The selected image / its URL is saved to the report definition **.repx** file.
+
+## Assign an Image from the Report's Image Collection
+
+Set the report's **Image Resources** property.
+
+![](../../../../../images/eurd-win-ImageResources-Editor.png)
+
+Invoke the **Picture Box**'s smart tag. Click the **Expression** option's ellipsis button to invoke the **Expression Editor**. Choose an image from the **Images** collection:
```

Comment:
"Invoke the **Picture Box**'s smart tag."
I don't think you can invoke a smart tag. Are you clicking the smart tag to invoke a context menu, possibly?

Reply:
I agree, thanks!

Reply:
So can we say "Click the **Picture Box** smart tag to invoke the context menu."